src/js/contexts/AnnounceContext/announce-context.stories.js

```diff
@@ -2,35 +2,32 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { storiesOf } from '@storybook/react';
 import { grommet } from 'grommet/themes';
-import { Box, Grommet, Text, AnnounceContext } from 'grommet';
+import { AnnounceContext, Box, Grommet, Heading, Text } from 'grommet';
 
 class Announcer extends Component {
   static propTypes = {
     announce: PropTypes.func.isRequired,
+    message: PropTypes.string,
+    mode: PropTypes.string,
+    role: PropTypes.string,
   };
-
-  constructor(props) {
-    super(props);
-    const { message, role, politenessSetting } = this.props;
-    this.state = {
-      message:
-        message || 'Here is a simple announcement. This will soon disappear',
-      politenessSetting: politenessSetting || 'polite',
-      role: role || 'log',
-    };
+  
+  static defaultProps = {
+    message: 'Here is a simple announcement. This will soon disappear',
+      mode: 'polite',
```

Comment:
indentation is off, how did this make it through `prettier`?

Reply:
Not sure. Fixed it.